roles/monit/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+- name: Setup monitrc file base
+  tags: setup_monitrc_base
+  become: true
+  blockinfile:
+    dest: "{{ monit_control_path }}"
+    create: yes
+    owner: root
+    group: root
+    mode: 0600
+    block: |
+      {{ monitrc_file_content }}
+
+- name: Prompt for SMTP input
```

Comment:
this wont work:
https://stackoverflow.com/questions/25466675/ansible-to-conditionally-prompt-for-a-variable

I need to instead just place this into a var file that will be read in and only run if the `setup_smtp` is set